.github/workflows/bridge.yml

```diff
@@ -0,0 +1,157 @@
+            environment-file: carsus/carsus_env3.yml
+            activate-environment: carsus
+            use-mamba: true
+
+      - name: Install package
+        run: pip install -e carsus/
+
+      - name: Run notebooks
+        run: |
+          ${{ env.NBCONVERT_CMD }} carsus/docs/refdata_gen.ipynb
+        env:
+          CARSUS_REFDATA: ${{ github.workspace }}/carsus-refdata
+        
+      - name: Upload Atom Data
+        uses: actions/upload-artifact@v2
+        with:
+          name: atom-data
+          path: carsus/docs/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5
+  
+  tardis-build:
+    needs: carsus-build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: tardis-sn/tardis
+          
+      - uses: actions/checkout@v2
+        with:
+          repository: atharva-2001/tardis-refdata
+          ref: raise_error_refdata_compare
+          path: tardis-refdata
+          lfs: true
+        
+      - name: Fetch lfs objects
+        run: |
+             cd tardis-refdata
+             git lfs checkout 
+
+      - name: Setup TARDIS environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            activate-environment: tardis
+            use-mamba: true
+
+      - uses: actions/cache@v2
+        with:
+          path: /usr/share/miniconda3/envs/tardis
+          key: conda-linux-64-${{ hashFiles('conda-linux-64.lock') }}-${{ env.CACHE_NUMBER }}
+        id: cache-conda
+
+      - name: Update environment
+        run: mamba update -n tardis --file conda-linux-64.lock
+        if: steps.cache-conda.outputs.cache-hit != 'true'
+
+      - name: Install bokeh
+        run: mamba install bokeh=2.2 --channel conda-forge --no-update-deps --yes
+
+      - name: Install package
+        run: pip install -e .
+        
+      - name: Download atom data
+        uses: actions/download-artifact@v2
+        with:
+          name: atom-data
+          path: carsus/docs/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5      
+      
+      - name: Replace Atomic Data
+        run: cp carsus/docs/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5/kurucz_cd23_chianti_He_cmfgen_H_Si_I-II.h5 tardis-refdata/atom_data/kurucz_cd23_chianti_H_He.h5
+      
+      - name: Generate Reference Data Copy
+        run: cp tardis-refdata/unit_test_data.h5 tardis-refdata/unit_test_data_org.h5 
+
+      - name: Generate New Reference Data
+        run:  pytest tardis ${{ env.PYTEST_FLAGS }}
+
+      - name: Compare Reference Data
+        run: |
+            ${{ env.NBCONVERT_CMD }} "tardis-refdata/notebooks/ref_data_compare_from_paths.ipynb"
+      
+      - name: Upload spectrum notebook
+        uses: actions/upload-artifact@v2
+        with:
+          path: "tardis-refdata/notebooks/ref_data_compare_from_paths.html"
+      
+      - name: Check if spectra were equal during comparision
```

Comment:
Does it only check if the spectra are equal or does it compare all of the ref_data? If the latter is the case, I would rename the step "Check if ref_data are equal" or something similar?

Reply:
Yes, it compares all the reference data and if it does not match, it changes the contents of a file in the `/tmp` folder. I renamed the step in commit: https://github.com/tardis-sn/carsus/pull/313/commits/9eb8aa8450dfa78d560064a0101537202bd2b34b.